src/component/tooltip/TooltipContent.js

```diff
@@ -169,7 +172,7 @@ function TooltipContent(container, api, opt) {
     var zr = this._zr = api.getZr();
     var appendToBody = this._appendToBody = opt && opt.appendToBody;
 
-    this._styleCoord = [0, 0];
+    this._styleCoord = [0, 0, 0];
```

Comment:
Looks good. But it still needs a ratio of `height` to handle the height change and relocate the tooltip.
So `_styleCoord` should be `[0, 0, 0, 0]`.